scripts/globals/effects/last_resort.lua

```diff
@@ -11,9 +11,12 @@ require("scripts/globals/status");
 -----------------------------------
 
 function onEffectGain(target,effect)
-    target:addMod(dsp.mod.ATTP,15 + target:getMerit(dsp.merit.LAST_RESORT_EFFECT));
-    target:addMod(dsp.mod.DEFP,-15 - target:getMerit(dsp.merit.LAST_RESORT_EFFECT));
-    target:addMod(dsp.mod.HASTE_ABILITY, 156+effect:getPower())
+    target:addMod(dsp.mod.ATTP, 15 + target:getMerit(dsp.merit.LAST_RESORT_EFFECT));
```

Comment:
it appears our script is on oldschool values. BG says its 25% in modern retail and we've got 15, which I remember being correct a long time ago. ffxiclopedia's page, which is old/not up to date, says 15, and also says it effects ranged attack as well. I dunno if that's true - can't say I ever cared about my drk's ranged attack.
```
 Increases attack by 25% (64/256) and reduces defense by 25% (64/256).
Effect can be increased to ~+34.77% (89/256) attack , ~-34.77% (89/256) defense with 5/5 Last Resort Effect Merits (~1.95% per level, 5/256).
```
IIiiiiiiiiiii hates fractional craps. Ignore the fractional-ness for the moment. Just another of those things we can't get perfect.

Reply:
The fractions would be easier if we actually did 256 base. It's done in some places and not others. For example, Smite. What's also funny is HASTE_ABILITY is 1024 base. XD